README.md

```diff
@@ -17,40 +14,137 @@ Through a series of exercises, we look at specific Scala 3 features and apply th
+
+```bash
+$ cmtc
+Usage: cmtc <COMMAND>
+
+Commands:
+  goto-exercise        Move to a given exercise. Pull in tests and readme files for that exercise
+  goto-first-exercise  Move to the first exercise. Pull in tests and readme files for that exercise
+  install              Install a course - from either a local directory, a zip file on the local file system or a Github project
+  list-exercises       List all exercises and their IDs in the repo. Mark the active exercise with a star
+  list-saved-states    List all saved exercise states, if any.
+  next-exercise        Move to the next exercise. Pull in tests and readme files for that exercise
+  previous-exercise    Move to the previous exercise. Pull in tests and readme files for that exercise
+  pull-solution        Pull in all code for the active exercise. All local changes are discarded
+  pull-template        Selectively pull in a given file or folder for the active exercise
+  restore-state        Restore a previously saved exercise state
+  save-state           Save the state of the active exercise
+  set-current-course   Sets the current course to point to a directory
+  version              Print version info
+```
+  
+### Install the course
+
+Installing this course can be done via the `cmtc install` command as follows:
+
+```bash
+$ cmtc install -f -s lunatech-labs/lunatech-scala2-to-scala3-course
+Downloading studentified course from 'https://github.com/lunatech-labs/lunatech-scala2-to-scala3-course/releases/download/3.0.2/lunatech-scala2-to-scala3-course-student.zip' to courses directory
+
+Project lunatech-scala2-to-scala3-course (3.0.2) successfully installed to:
+  /Users/ericloots/Library/Caches/com.lunatech.cmt/Courses/lunatech-scala2-to-scala3-course
+
+Current course set to '/Users/ericloots/Library/Caches/com.lunatech.cmt/Courses/lunatech-scala2-to-scala3-course'
+
+Exercises in repository:
+  1.  *   exercise_000_sudoku_solver_initial_state
+  2.      exercise_001_dotty_deprecated_syntax_rewriting
+  3.      exercise_002_dotty_new_syntax_and_indentation_based_syntax
+  4.      exercise_003_top_level_definitions
+  5.      exercise_004_parameter_untupling
+  6.      exercise_005_extension_methods
+  7.      exercise_006_using_and_summon
+  8.      exercise_007_givens
+  9.      exercise_008_enum_and_export
+ 10.      exercise_009_union_types
+ 11.      exercise_010_opaque_type_aliases
+ 12.      exercise_011_multiversal_equality
+ 13.      exercise_020_opaque_type_aliases_alt
+ 14.      exercise_021_multiversal_equality
+```
+
+As can be seen in the above output, `cmtc` installed the course in the `/Users/ericloots/Library/Caches/com.lunatech.cmt/Courses/lunatech-scala2-to-scala3-course` folder. Note that this location is OS specific, so, take a note of the location.
+
+### Loading the course exercises
+
+You can now load the course exercises in the development environment of your choice. You will be
```

Comment:
```suggestion
You can now load the course exercises in the development environment of your choice. You will be
```

What does it mean to load the course in the development environment of your choice?

Reply:
I mean the IDE of choice (IntelliJ or VSCode with metals, or ...)

Reply:
```suggestion
You can now load the course exercises in the development environment of your choice (IntelliJ, VSCode with Metals, etc). You will be
```